docs/reference/object-detection-2d-nanodet.md

```diff
@@ -160,6 +160,34 @@ Parameters:
 - **lazy_load**: *bool, default=True*\
   Enables loading optimized model from predetermine path without export it each time.
 
+#### `NanodetLearner.optimize_c_model`
+```python
+NanodetLearner.optimize_c_model(self, export_path, conf_threshold, iou_threshold, nms_max_num, hf, dynamic, verbose)
+```
+
+This method is used to export a JIT optimized model with its metadata compatible with the C API.
+If a model is already present in the *export_path* it will be replaced.
+Inside this folder, the model is saved as *nanodet_{model_name}.pth*
+and a metadata file *nanodet_{model_name}.json*.
+
+Parameters:
+
+- **export_path**: *str*\
+  Specifies the path to save the optimized model.
+- **conf_threshold**: *float, default=0.35*\
+  Specifies the threshold for object detection inference.
+  An object is detected if the confidence of the output is higher than the specified threshold.
+- **iou_threshold**: *float, default=0.6*\
+  Specifies the IOU threshold for NMS in inference.
+- **nms_max_num**: *int, default=100*\
+  Determines the maximum number of bounding boxes that will be retained following the nms.
```

Comment:
No, because post processing is part of the export model I think it is better for the user to add them each time so they do not use something like the "defaults". I think that your comment is better than have some default values that change the output of C script.

Reply:
If you still think that it would be better to have some default values I will change it accordingly.

Reply:
Ok if you think it's better for the user to specify these values, i firstly suggest to remove the `default` entries from the docs as in the suggestion, and secondly to add some text in the description below each argument to give some guidelines on how to set them. For example:

```
- **conf_threshold**: *float*\
  Specifies the threshold for object detection inference. The value needs to be set between 0.1 and 1.5, modify to achieve best results.
```